db/migrate/20231111183151_commentsnull.rb

```diff
@@ -0,0 +1,11 @@
+class Commentsnull < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
```

Comment:
- [ ] Please fix this migration error because you already have a comments table. I can't run the server because of this error. 
![image](https://github.com/sheezu900/blog-rails/assets/91486791/1a4edd4a-a412-4289-bf0a-7fd8c282c651)

![image](https://github.com/sheezu900/blog-rails/assets/91486791/b291cff1-eaa8-4286-a768-6a23f67027a8)